platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/export/ConfigExport.java

```diff
@@ -66,23 +73,20 @@ public static void printConfig(final Configuration configuration, final Consumer
                 .filter(name -> !recordProperties.containsKey(name))
                 .forEach(name -> nonRecordProperties.put(name, configuration.getValue(name)));
 
-        final Set<String> allConfigNames = combine(recordProperties.keySet(), nonRecordProperties.keySet());
         final Set<Object> allConfigValues = combine(recordProperties.values(), nonRecordProperties.values());
-
-        final int maxNameLength = getMaxPropertyLength(allConfigNames);
         final int maxValueLength = getMaxPropertyLength(allConfigValues);
 
         // Write all record defined values first, in alphabetical order
         recordProperties.keySet().stream().sorted().forEach(name -> {
             final Object value = recordProperties.get(name);
-            final String line = buildLine(name, value, maxNameLength, maxValueLength, "  [USED IN RECORD]");
+            final String line = buildLine(name, value, maxValueLength, "");
             lineConsumer.accept(line);
         });
 
         // Write all values not defined in records next, in alphabetical order
         nonRecordProperties.keySet().stream().sorted().forEach(name -> {
             final Object value = nonRecordProperties.get(name);
-            final String line = buildLine(name, value, maxNameLength, maxValueLength, "  [NOT USED IN RECORD]");
+            final String line = buildLine(name, value, maxValueLength, "  [NOT USED IN RECORD]");
```

Comment:
Do we still need `[NOT USED IN RECORD]`? Is it even possible to define configuration without using a record? Can this entire block of code be deleted?

Reply:
You could define a value with is not used in any. record and accesses via `com.swirlds.config.api.Configuration#getValue(java.lang.String)` and then you will find 

```
someRandomSetting, 123                      [NOT USED IN RECORD]
```